### ISSUE TITLE
How?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-aquaponics#2]
Watch the videos:
- Basic intro: https://youtu.be/niJAAMlf08E and https://youtu.be/TELxeg-WI4k (part 2)
- Chanel: https://www.youtube.com/user/aquaponicgardening
- GeoThermal Heating and Cooling in Aquaponics: https://youtu.be/7V2NSnw8Tws
- High Tech Aquaponics Program: https://youtu.be/DmrET5XF2Z8

Read the book:
https://www.amazon.co.uk/Aquaponic-Gardening-Step-Step-Vegetables/dp/1908643080

Sites:
- http://www.canna-uk.com/aquaponics
- British Aquaponics Association: http://www.baqua.org.uk/ 
- shop: http://shopaquaponics.co.uk/
- FishPlant: http://www.fishplant.co.uk/ and http://www.fishplant.co.uk/index.php?q=node/53 (_shop_)

**Comment by nelsonic** [comment]
+ British Aquaponics Association: http://www.baqua.org.uk/ 
+ shop: http://shopaquaponics.co.uk/

**Comment by nelsonic** [comment]
Do fish need sunlight? http://www.backyardaquaponics.com/guide-to-aquaponics/locating-your-system/

**Comment by nelsonic** [comment]
@eliascodes https://www.seneye.com/ 😉 
which I discovered on: https://www.seneye.me/BristolFish#p=32604
that I got to from: https://bristolfish.org/about/

**Comment by eliasmalik** [comment]
@nelsonic 👍 

A lot of the features you'd like. Would probably be a slam dunk if it could interface with an Arduino to close a control loop. Says it comes with a USB cable, but can't find what you can actually do with it!

**Comment by nelsonic** [comment]
Yeah, as soon as we've sorted out https://github.com/dwyl/phase-two/issues/11 we will _allocate_ some time/funds to this. https://youtu.be/8A7TOjlIO2o 😉
